public/app/common/servcies/devices/discovery-service/discovery-service.js

```diff
@@ -6,16 +6,18 @@ var SessionState = require('./discovery-session-state.js');
 var startDiscoveryUrl = 'http://householdmockapi000.yetudev.com:8080/gateway/discoveries';
 var gatewayStatusUrl = 'http://householdmockapi000.yetudev.com:8080/gateway';
 
-var subject = new Rx.Subject();
-var $pollInterval = Rx.Observable
-  .interval(200)
-  .takeUntil(subject);
 
 module.exports = {
 
   startDiscovery: function startDiscovery () {
+    var subject = new Rx.Subject();
```

Comment:
If it does not receive and emit values, it can be an `Observable` with `fromCallback` call